docs/scos/dev/glue-api-guides/202311.0/glue-api-infrastructure/backend-and-storefront-api-module-differences.md

```diff
@@ -8,11 +8,12 @@ redirect_from:
   - /docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/backend-and-storefront-api-module-differences.html
   - /docs/scos/dev/glue-api-guides/202212.0/decoupled-glue-infrastructure/backend-and-storefront-api-module-differences.html
   - /docs/scos/dev/glue-api-guides/202311.0/decoupled-glue-infrastructure/backend-and-storefront-api-module-differences.html
+  - /docs/scos/dev/glue-api-guides/202311.0/backend-and-storefront-api-module-differences.html
 ---
 
 This document describes differences between the backend and storefront code in API modules.
 
-The Glue infrastructure can build not only the storefront but also backend APIs in the same project. *storefront APIs* inherit the possibilities of the old Spryker Glue implementation in terms of what they can do and what they have access to. For example, storefront APIs can access Storage and Elasticsearch and make RPC Zed calls using Clients. *backend APIs* have direct access to Facades, which lets you create performant backend APIs that you might often need for your projects.
+The Glue infrastructure can build not only the storefront but also backend APIs in the same project. *storefront APIs* inherit the possibilities of the Legacy Spryker Glue implementation in terms of what they can do and what they have access to. For example, storefront APIs can access Storage and Elasticsearch and make RPC Zed calls using Clients. *backend APIs* have direct access to Facades, which lets you create performant backend APIs that you might often need for your projects.
```

Comment:
"For example, storefront APIs can access Storage and Elasticsearch and make RPC Zed calls using Clients." 
have we not switched from Elasticsearch to OpenSearch? 

"...that you might often need for your projects." Why? please and examples and references.